tapdb/universe.go

```diff
@@ -323,9 +324,15 @@ func (b *BaseUniverseTree) RegisterIssuance(ctx context.Context,
 	defer b.registrationMtx.Unlock()
 
 	dbErr := b.db.ExecTx(ctx, &writeTx, func(dbTx BaseUniverseStore) error {
-		issuanceProof, _, err = universeRegisterIssuance(
-			ctx, dbTx, b.id, key, leaf, metaReveal,
-		)
+		if replace {
+			issuanceProof, _, err = universeReplaceIssuance(
+				ctx, dbTx, b.id, key, leaf,
+			)
+		} else {
+			issuanceProof, _, err = universeRegisterIssuance(
```

Comment:
Alternatively, Universe servers could have a rule where they only accept assets that have been sufficiently buried. This wouldn't help with the multi-verse case though, when you want to send a proof to someone as soon as it's been confirmed.

Reply:
Hmm, we could add that restriction exclusively for issuance proofs. But then if you mint and want to send immediately, you would be blocked by that restriction I guess.